src/test/unit/services/JobRoleServicesTest.ts

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import { expect } from 'chai';
+import { JobRoleResponse } from "../../../models/JobRoleResponse";
+import { describe } from "node:test";
+import { getJobRoles, URL} from '../../../services/JobRoleService';
+
+const expected: JobRoleResponse = {
+    roleName: "Manager",
+    location: "New York", 
+    capability: "Coding",
+    band: "B",
+    closingDate: new Date(2019, 1, 16)
+}
+const mock = new MockAdapter(axios);
+
+describe('JobRoleService', function () {
+    describe('getJobRoles', function () {
+        it('should return roles from response', async () => {
+            const data = [expected];
+            mock.onGet(URL).reply(200,data);
+            const results = await getJobRoles();
+
+
+            const actual: JobRoleResponse = {
+                roleName: results[0].roleName,
+                location: results[0].location, 
+                capability: results[0].capability,
+                band: results[0].band,
+                closingDate: new Date(results[0].closingDate)
+
+            }
```

Comment:
you shouldn't need to do this. just check if results[0] deep equals expected

Reply:
We had that originally, but it was failing because it was trying to compare a string to a date object. This was our way to fix that

Reply:
don't think you'll need to do this if you set your test data up correctly, or just use a string for the date